sympy/printing/pretty/pretty.py

```diff
@@ -1522,6 +1521,30 @@ def _print_Contains(self, e):
         else:
             return prettyForm(sstr(e))
 
+    def _print_SeqFormula(self, s):
```

Comment:
`_print_SeqBase` could be used here instead - then you wouldn't need the additional methods for `SeqAdd`, etc...